### PR TITLE
Navbar, Filters, Added products to database

### PR DESCRIPTION
**Pull request by AnuragSahu11**
*Made navbar fully responsive
*Made filter section responsive
*Added  products data of category investment, charting to database
*Made small changes to product cards in product listing, wishlist and cart section

### CODE REVIEW

**Review by GuptaEesh — COMMENTED** [review]
Code structure looks good.
Explanatory-variable name.
Could have used switch instead of if-else.
Otherwise,good to merge

File: lucid-book-store/public/index.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
     -->
     <title>React App</title>
```
Comment (junaidshaikh-js) [inline lucid-book-store/public/index.html:27]: Give good title name

**Review by junaidshaikh-js — APPROVED** [review]
Great work! 

Clean folder structure. Next time I would suggest raising PR for smaller file changes as a large number of files are not suitable for review at a single time and the reviewer might miss some points.

File: lucid-book-store/public/index.html (hunk 1 above)
Comment (AnuragSahu11) [reply lucid-book-store/public/index.html:27]: Okay I will raise PR for smaller changes, I will also change the title

**Pull request merged**